bucket/plantuml.json

```
@@ -13,7 +13,7 @@
     "url": "https://downloads.sourceforge.net/project/plantuml/1.2020.22/plantuml.1.2020.22.jar#/plantuml.jar",
     "hash": "sha1:1ece779643e06549e3e76922898314cbaea07e6a",
     "bin": "plantuml.cmd",
-    "pre_install": "\"@java -jar \"\"$dir\\plantuml.jar\"\" -graphvizdot \"\"$(shimdir $global)\\dot.exe\"\" %*\" | out-file -en oem \"$dir\\plantuml.cmd\"",
+    "pre_install": "\"@java -jar \"\"$dir\\plantuml.jar\"\" %*\"\" -graphvizdot \"\"$(shimdir $global)\\dot.exe\" | out-file -en oem \"$dir\\plantuml.cmd\"",
```

Nonsense to put escape single -graphvizdot wihtout the path, which needs to be escaped and it is not

```suggestion
    "pre_install": "\"@java -jar \"\"$dir\\plantuml.jar\"\" %* -graphvizdot \"\"$(shimdir $global)\\dot.exe\"\"\" | out-file -en oem \"$dir\\plantuml.cmd\"",
```